apiserver/admin.go

```diff
@@ -160,8 +160,11 @@ func (a *admin) getAuditRecorder(req params.LoginRequest, authResult *authResult
 	if !authResult.userLogin || a.srv.auditLogger == nil {
 		return nil, nil
 	}
+	// Wrap the audit logger in a filter that prevents us from logging
+	// lots of readonly conversations (like juju status requests).
```

Comment:
nit - I'd prefer to surround 'juju status' in quotes :)

Reply:
Done!